routes/users.js

```diff
@@ -33,7 +33,7 @@ usersRouter.post('/register', async (req, res, next) => {
 				user: newUser.username,
 				isAdmin: newUser.isAdmin
 			},
-			JWT_SECRET
+			process.env.JWT_SECRET
```

Comment:
The secret was already destructured up top

Reply:
I know, I saw that and tried to keep it as is, but for some reason I kept getting a "missingsecretkey" error when I kept it as is. I just changed it to circumvent that error, which it did. We maybe be able to change it back no problem, not sure what the origin of my problem with that piece was still. Good catch as well!